### PR TITLE
Created BigPicture and LittlePicture scaffolding

### PR DESCRIPTION
**Pull request by jayajr**

### CODE REVIEW

File: app/controllers/big_pictures_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  end
+
+  # POST /big_pictures
+  # POST /big_pictures.json
+  def create
+    @big_picture = BigPicture.new(big_picture_params)
+
+    respond_to do |format|
+      if @big_picture.save
+        format.html { redirect_to @big_picture, notice: 'Big picture was successfully created.' }
```
Comment (djtran) [inline app/controllers/big_pictures_controller.rb]: Will we be able to use these methods without changing the current page a user can see? Does the redirect forcefully change the view?

File: app/views/big_pictures/_big_picture.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1,2 @@
+json.extract! big_picture, :id, :uuid, :name, :description, :flavor, :location, :time, :culture, :edges, :created_at, :updated_at
```
Comment (djtran) [inline app/views/big_pictures/_big_picture.json.jbuilder]: What does this do? Does this just show a plaintext json object to the user? looks useful

File: app/views/big_pictures/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<%= form_with(model: big_picture, local: true) do |form| %>
```
Comment (djtran) [inline app/views/big_pictures/_form.html.erb]: Will it be possible to take this form and place it within other .html.erb files? Can we use this as a component and insert it into some of Julie's UIs? 

nice nice nice nice if that's the case

File: app/controllers/big_pictures_controller.rb (hunk 1 above)
Comment (jayajr) [reply app/controllers/big_pictures_controller.rb]: If I understand it right, the format.___ functions return the stuff you requested. format.json is used for API calls, so if we don't need the format.html, we just remove it

File: app/views/big_pictures/_form.html.erb (hunk 3 above)
Comment (jayajr) [reply app/views/big_pictures/_form.html.erb]: Yes! Just plug  `_form.html.erb` into the views folder, and throw in a `<%= render 'form', big_picture: @big_picture %>` in the view. I *think* you can also just rename `_form` to `_b_p_form` and then `render 'b_p_form'` in the view if you need multiple forms

File: app/views/big_pictures/_big_picture.json.jbuilder (hunk 2 above)
Comment (jayajr) [reply app/views/big_pictures/_big_picture.json.jbuilder]: I think this is related to the format.json in the controller, and builds the json from the model

**Pull request merged**